UserManagement/src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+#Local MongoDB config
+spring.data.mongodb.port=27017
+spring.data.mongodb.host=localhost
+spring.data.mongodb.database=admin
```

Comment:
Kindly remove the .jar file from the commit.

Reply:
Got it!